.github/workflows/linter.yml

```diff
@@ -91,4 +91,5 @@ jobs:
           VALIDATE_TYPESCRIPT_ES: true
 
           # Flag to enable or disable the linting process of the YAML language.
-          VALIDATE_YAML: true
+          # Note - disabled as of 26 Jan, 2022
+          # VALIDATE_YAML: true
```

Comment:
Not a blocker - curious why this is needed here (found YAML validation being turned off around that time in Prospect API: https://github.com/Pocket/prospect-api/pull/280, but it's still there in the backend template). 

If the linter is still being buggy after 12+ months, maybe we should remove these lines entirely? They're omitted from the linter setup in one of the newest repos (Shareable Lists API).

Reply:
thanks Nina! we have a follow-up PR to add some curation fields like topics, will add to that PR. will make merge this to test in prod